cmd/krel/cmd/push.go

```diff
@@ -152,51 +161,56 @@ func init() {
 }
 
 func runPushBuild(opts *pushBuildOptions) error {
-	log.Println("unimplemented")
-
 	var latest string
 	releaseKind := opts.releaseKind
 
 	// Check if latest build uses bazel
 	dir, err := os.Getwd()
 	if err != nil {
-		log.Fatal(err)
+		logrus.Errorf("Unable to get working directory: %v", err)
+		return err
 	}
 
 	isBazel, err := release.BuiltWithBazel(dir, releaseKind)
 	if err != nil {
-		log.Fatal(err)
+		logrus.Errorf("Unable to identify if release built with Bazel: %v", err)
+		return err
 	}
 
 	if isBazel {
-		log.Println("Using Bazel build version.")
+		logrus.Info("Using Bazel build version")
 		version, err := release.ReadBazelVersion(dir)
 		if err != nil {
-			log.Fatal(err)
+			logrus.Errorf("Unable to read Bazel build version", err)
+			return err
 		}
 		latest = version
 	} else {
-		log.Println("Using Dockerized build version.")
+		logrus.Info("Using Dockerized build version")
 		version, err := release.ReadDockerizedVersion(dir, releaseKind)
 		if err != nil {
-			log.Fatal(err)
+			logrus.Errorf("Unable to read Dockerized build version: %v", err)
+			return err
 		}
 		latest = version
 	}
 
-	log.Println("Found build version: " + latest)
+	logrus.Info("Found build version: " + latest)
```

Comment:
I would prefer `Infof` and `%s` but this is just a nit.

Reply:
Ah yes I agree, thanks for pointing this out!